app-backend/api/src/main/scala/grid/QueryParameters.scala

```diff
@@ -27,12 +27,12 @@ trait GridQueryParameterDirective extends QueryParametersCommon
     'maxSunElevation.as[Float].?,
     'minSunElevation.as[Float].?,
     'ingested.as[Boolean].?
-  )).as(GridQueryParameters)
+  )).as(GridQueryParameters.apply _)
```

Comment:
Is this necessary if you use the annotated generation of codecs? I'm assuming this was a workaround based on needing a companion object to stuff codecs into